src/main/java/org/myrobotlab/framework/interfaces/ServiceInterface.java

```diff
@@ -138,7 +138,17 @@ public interface ServiceInterface extends ServiceQueue, LoggingSink, NameTypePro
    * 
    * @param config
    */
-  void setConfig(ServiceConfig config);
+  ServiceConfig setConfig(ServiceConfig config);

```

Comment:
why return ServiceConfig here?

Reply:
https://medium.com/miro-engineering/fluent-setter-breaking-the-convention-33ce3433126e

Reply:
set to void